Cargo.toml

```diff
@@ -15,7 +15,7 @@ license = "BSD-3-Clause"
 maintenance = { status = "experimental" }
 
 [dependencies]
-winapi = { version = "0.3", features = ["bcrypt", "ntstatus"] }
+winapi = { version = "0.3", features = ["bcrypt", "ntstatus", "winerror", "winbase", "debug"] }
```

Comment:
Is `debug` really required?

Reply:
It's for `Debug` impl for winapi structs, IIRC I needed that when I was debugging something but we definitely not should unconditionally enable it